internal/provider/provider.go

```diff
@@ -3,6 +3,7 @@ package provider
 import (
 	"context"
 	"fmt"
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/service/dns_config"
```

Comment:
Please fix import ordering.

This should go after the hashicorp imports and please add this to infobloxopen import group

Reply:
done